NetCord/Embed.cs

```diff
@@ -1,24 +1,83 @@
 ﻿namespace NetCord;
 
+/// <summary>
+/// Displays embedded content such as an image or URL, alongside a title and various other fields. All text in the embed must add up to under 6000 characters, and you can only have up to 10 embeds per message.
+/// </summary>
 public class Embed : IJsonModel<JsonModels.JsonEmbed>
 {
     JsonModels.JsonEmbed IJsonModel<JsonModels.JsonEmbed>.JsonModel => _jsonModel;
     private readonly JsonModels.JsonEmbed _jsonModel;
-
+    
+    /// <summary>
+    /// The text that is placed above the description, usually highlighted. Also directs to a URL if given, has a 256 character limit.
+    /// </summary>
+    
     public string? Title => _jsonModel.Title;
+    
+    /// <summary>
+    /// The type of the embed, always set to <see cref="EmbedType.Rich"> for webhook embeds. </see>.
+    /// </summary>
     public EmbedType? Type => _jsonModel.Type;
+    
+    /// <summary>
+    /// The part of the embed where most of the text is contained, limited to 4096 characters.
```

Comment:
Most of the text can be placed into fields too, I am not sure if the doc is ok

Reply:
Well in this case that's just what discord's documentation said, I did find it weird though.

Reply:
I think discord's documentation says:
> description of embed

Reply:
![image](https://github.com/KubaZ2/NetCord/assets/77853483/4293a683-5f88-41a3-a15a-4bca9cc2e154)

Reply:
![image](https://github.com/KubaZ2/NetCord/assets/114494949/85dfb642-6cf0-4644-826d-4f3e0f520715)
Though reading it again this is really inaccurate at some points (such as the timestamp thing which I seemingly missed)_

Reply:
What is it? It doesn't seem to be the Discord official documentation

Reply:
https://discord.com/safety/using-webhooks-and-embeds#:~:text=your%20preferred%20channel!-,What%20is%20an%20Embed%3F,-We%E2%80%99ve%20been%20talking

Reply:
The official embed documentation is here: https://discord.com/developers/docs/resources/channel#embed-object-embed-structure, but well, if Discord posted it, it should be ok 😁

Reply:
Yeah I saw the official embed documentation but honestly it was about the equivalent of:

The color red
Red